doc/source/install/index.rst

```diff
@@ -0,0 +1,74 @@
+.. _install:
+
+=================================
+Installing Percona XtraDB Cluster 
+=================================
+
+|PXC| supports most 64-bit Linux distributions.
+Percona provides packages for popular DEB-based and RPM-based distributions:
+
+* Debian 7 ("wheezy")
+* Debian 8 ("jessie")
+* Ubuntu 12.04 LTS (Precise Pangolin)
+* Ubuntu 14.04 LTS (Trusty Tahr)
+* Ubuntu 15.10 (Wily Werewolf)
+* Ubuntu 16.04 (Xenial Xerus)
+* Red Hat Enterprise Linux / CentOS 6
+* Red Hat Enterprise Linux / CentOS 7
+
+Prerequisites
+=============
+
+|PXC| requires the following ports for communication: 3306, 4444, 4567, 4568.
+Make sure that these ports are not blocked by firewall
+or used by other software.
+
+|PXC| does not work with ``SELinux`` and ``AppArmor`` security modules.
+Make sure these modules are disabled.
+
+There are several MySQL configuration options
+that are mandatory for running |PXC|.
+These include the path to the Galera wsrep library,
+connection URL for other nodes in the cluster,
+and method used for state snapshot transfer (SST).
+Make sure that the binlog format is set to ``ROW``,
+the default storage engine is InnoDB (MyISAM has only experimental support),
+and set the mode of InnoDB autoincrement locks to ``2``.
+The following is an example of the :file:`my.cnf` file:
+
+.. code-block:: none
+
+   [mysqld]
+
+   wsrep_provider=/usr/lib64/libgalera_smm.so
+   wsrep_cluster_address=gcomm://10.11.12.206
```

Comment:
server-id is not more default in MySQL-5.7.
If user want to use log-bin then server-id has to be explicitly specified.

log-bin=mysql-bin
# server id should be unique for each node of pxc-cluster
server-id=1